src/DynamoCore/DynamoCore.csproj

```diff
@@ -346,7 +345,6 @@ limitations under the License.
     <ProjectReference Include="..\NodeServices\DynamoServices.csproj">
       <Project>{ef879a10-041d-4c68-83e7-3192685f1bae}</Project>
       <Name>DynamoServices</Name>
-      <Private>False</Private>
```

Comment:
Is this intentional, I suspect VS silently removed it?

Reply:
Yes, it's unintentional.